sdk/wallet-adapter/example/src/App.tsx

```diff
@@ -2,14 +2,14 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import "./App.css";
-import { WalletKitProvider, ConnectButton } from "@mysten/wallet-kit";
+import { WalletKitProvider, ConnectButton, useWalletKit } from "@mysten/wallet-kit";
 
 function App() {
+  const { currentWallet } = useWalletKit();
+  console.log(currentWallet);
```

Comment:
Did you leave this console here intensionally?

Reply:
No but this is a demo app so it's probably fine, but I'll remove it.